config/webhook/manifests.yaml

```diff
@@ -162,7 +162,7 @@ webhooks:
     - CREATE
     - UPDATE
     resources:
-    - redises
+    - redis
```

Comment:
This seems to still need to be plural, given that the webhook isn't firing in the CI tests [1].

[1] https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openstack-k8s-operators_infra-operator/503/pull-ci-openstack-k8s-operators-infra-operator-main-infra-operator-build-deploy-kuttl/1995942843947945984